.circleci/install_validator_dependencies.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+
+# Script used by AMP's CI builds to install AMP Validator dependencies on
+# CircleCI.
+
+set -e
+
+GREEN() { echo -e "\033[0;32m$1\033[0m"; }
+
+echo $(GREEN "Adding Bazel repo...")
+sudo apt install curl gnupg
```

Comment:
From the [logs](https://app.circleci.com/pipelines/github/ampproject/amphtml/5301/workflows/dfa59126-ab83-4fc1-89d0-a10287738cd3/jobs/72058/parallel-runs/0/steps/0-108), it looks like neither `curl` nor `gnupg` need to be explicitly installed. Looks like this line can be removed.

![image](https://user-images.githubusercontent.com/26553114/112190239-32202c00-8bdb-11eb-8733-7ad0e4e2232a.png)

Reply:
Done